scripts/quests/jeuno/Apocalypse_Nigh.lua

```diff
@@ -0,0 +1,290 @@
+-----------------------------------
+-- Apocalypse Nigh
+-----------------------------------
+-- !addquest 3 89
+-- Ru'Lude Gardens Palace Region 1
+-----------------------------------
+require('scripts/globals/interaction/quest')
+require('scripts/globals/common')
+require('scripts/globals/items')
+require("scripts/globals/npc_util")
+require('scripts/globals/quests')
+require('scripts/globals/titles')
+require('scripts/globals/zone')
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.JEUNO, xi.quest.id.jeuno.APOCALYPSE_NIGH)
+
+quest.reward =
+{
+    title    = xi.title.BREAKER_OF_THE_CHAINS,
+}
+
+quest.possibleItems =  {xi.items.STATIC_EARRING, xi.items.MAGNETIC_EARRING, xi.items.HOLLOW_EARRING, xi.items.ETHEREAL_EARRING}
```

Comment:
This could most likely just stay as a local instead of attaching it to the quest table, also would prefer that its broken out instead of one line